0.wave1_data/1.cellprofiler_ic_processing/scripts/run_loaddata_and_ic.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Create LoadData csv and run CellProfiler `illum.cppipe` pipeline - Wave 1
+
+# ## Import libraries
+
+# In[1]:
+
+
+import os
```

Comment:
Was this import used? If not, consider removing.

Reply:
👍 Good catch!